src/library.js

```diff
@@ -27,12 +27,12 @@ LibraryManager.library = {
 
   getTempRet0__sig: 'i',
   getTempRet0: function() {
-    return {{{ makeGetTempRet0() }}};
+    return getTempRet0();
   },
```

Comment:
this basically makes `_getTempRet0` forward the call to `getTempRet0`. Could we just mark this method as `$getTempRet0` (unprefixed), and include the body here? (moving it from `support.js`)

But this PR looks safe and good as it is, and that can be left for later.

Reply:
Yes, indeed, my understanding that the library functions `_getTempRet0` and `_getTempRet1` are nothing but wrappers about the runtime functions.. in order to allow the runtime functions to be used by native code (native code can't link against runtime functions directly).

I think removing the runtime versions from support.js might cause issue with external code that expects to be able to call the unprefixed versions.

Reply:
Yes, sounds like merging the prefixed and unprefixed versions would be nontrivial.